packages/salesforcedx-vscode-core/package.json

```diff
@@ -25,9 +25,10 @@
     "Other"
   ],
   "dependencies": {
-    "@salesforce/core": "2.4.1",
+    "@salesforce/core": "2.5.1",
```

Comment:
The Apex library is currently using version `2.5.1` of `@salesforce/core` and it seems like there were changes between versions `2.4.1` and `2.5.1` that are making the two versions incompatible w each other. If we update the dependency in VS Code to be `2.5.1`, we'll also have to update the version of the SDR library we're currently using to be the latest (currently unreleased) version since that has the correct version of `@salesforce/core`.

Reply:
Or, we can move the version of `@salesforce/core` used in the Apex library to `2.4.1` until the rest of these changes can made. thoughts?

Reply:
If salesforcedx-apex is not released yet I'd recommend changing the version to `2.4.1` so that it doesn't block this PR, unless we release new version of SDR library sooner

Reply:
Keep version 2.5.1, we'll be releasing a new version of deploy retrieve tomorrow, we'll just have to hold off on merging this PR so it doesn't break the build